source/NVDAObjects/IAccessible/ia2TextMozilla.py

```diff
@@ -324,11 +324,12 @@ def _getText(self, withFields, formatConfig=None):
 				field = controlStack.pop()
 				if field:
 					fields.append(textInfos.FieldCommand("controlEnd", None))
-					if ti.compareEndPoints(self._makeRawTextInfo(obj, textInfos.POSITION_ALL), "endToEnd") == 0:
+				if ti.compareEndPoints(self._makeRawTextInfo(obj, textInfos.POSITION_ALL), "endToEnd") == 0:
+					if field:

```

Comment:
Is there any case where field might be falsy but we also want to recognise the end of the node? For someone less than familiar with this code, how would we go about checking this?